test/proposals/ListOnZoraProposalIntegration.t.sol

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8;
+
+import "forge-std/Test.sol";
+
+import "../../contracts/globals/Globals.sol";
+import "../../contracts/globals/LibGlobals.sol";
+import "../../contracts/proposals/ProposalExecutionEngine.sol";
+import "../../contracts/distribution/TokenDistributor.sol";
+
+import "../TestUtils.sol";
+import "../DummyERC721.sol";
+import "./ZoraTestUtils.sol";
+import "../TestUsers.sol";
+
+contract ListOnZoraProposalIntegrationTest is Test, TestUtils, ZoraTestUtils {
```

Comment:
We've moved to have all test that utilizes forks be in contracts suffixed with `ForkedTest`

Reply:
There is already a `ListOnZoraProposalForked.t.sol` lol

Reply:
This test was integrated into `ListOnZoraProposalForked.t.sol` in #285